### DISCUSSION TITLE
PCMCI+ and undirected Links

### DISCUSSION THREAD

**Discussion by vinni94** [jakobrunge/tigramite#297] [General]
Hi, 
I am trying to do PCMC+ analysis over my time series data. I am getting two kinds of undirected links, one with rounded edges 0-----0 and the other with crossed edges x----x. How should I interpret such links? What is the difference between both of the links?

**Comment by sagarsimha** [comment]
Hi,
The tutorial below has description of these types of orientations,
https://github.com/jakobrunge/tigramite/blob/master/tutorials/causal_discovery/tigramite_tutorial_pcmciplus.ipynb

**Reply by vinni94** [reply]
Thanks much Sagar. Got it.
